packages/wonder-blocks-form/src/components/checkbox-group.tsx

```diff
@@ -130,43 +130,44 @@ const CheckboxGroup = React.forwardRef(function CheckboxGroup(
     const allChildren = React.Children.toArray(children).filter(Boolean);
 
     return (
-        <StyledFieldset data-testid={testId} style={styles.fieldset} ref={ref}>
-            {/* We have a View here because fieldset cannot be used with flexbox*/}
-            <View style={style}>
```

Comment:
This is an old implementation created ~6 years ago. This `flexbox` issue is no longer happening in current browser versions.